requirements/static/ci/common.in

```diff
@@ -18,6 +18,7 @@ etcd3-py==0.1.6 ; python_version >= '3.6'
 gitpython>=3.1.30; python_version >= '3.7'
 jmespath
 jsonschema
+junit_xml>=1.9
```

Comment:
Shouldn't this be  `junit-xml`?

Reply:
Yes, please change to what @twangboy said. Even if its named with an underscore on PyPi.